app/src/main/java/ar/edu/unlam/mobile/scaffolding/data/remote/interceptor/AuthInterceptor.kt

```diff
@@ -1,5 +1,33 @@
 package ar.edu.unlam.mobile.scaffolding.data.remote.interceptor
 
-class AuthInterceptor {
-    // TODO: Implementar interceptor para Application-Token
-}
+import ar.edu.unlam.mobile.scaffolding.data.remote.util.ApiConfig
+import okhttp3.Interceptor
+import okhttp3.Response
+import javax.inject.Inject
+
+class AuthInterceptor
+    @Inject
+    constructor(
+        private val tokenManager: TokenManager,
+    ) : Interceptor {
+        override fun intercept(chain: Interceptor.Chain): Response {
+            val request = chain.request()
+            val requestBuilder =
+                request
+                    .newBuilder()
+                    .header(
+                        ApiConfig.APPLICATION_TOKEN_HEADER,
+                        ApiConfig.APPLICATION_TOKEN_VALUE,
```

Comment:
```suggestion
                        BuildConfig.API_TOKEN,
```
Usemos el token de las local properties, no deberiamos tener una key de una api hardcodeada en el codigo. Se me paso mencionarlo en el PR que lo sumaste, pero lo podrias borrar en este PR?